engine/src/flutter/impeller/entity/contents/text_contents.cc

```diff
@@ -105,7 +105,9 @@ void TextContents::ComputeVertexData(
   size_t bounds_offset = 0u;
   for (const TextRun& run : frame->GetRuns()) {
     const Font& font = run.GetFont();
-    Scalar rounded_scale = TextFrame::RoundScaledFontSize(scale);
+    Scalar rounded_scale = frame->GetScale();
+    const Matrix transform =
+        frame->GetTransform() * Matrix::MakeTranslation(offset);
```

Comment:
Should this be `frame->offset` to match `TypographerContextSkia::CollectNewGlyphs`?

Reply:
Done